guide/misc/release-notes.md

```diff
@@ -14,19 +14,41 @@ to Brooklyn's commercial users for funding much of this development.
 
 ### New Features
 
+1. Auto-bundling of BOMs and the ability to manage and persist bundles:  related entities are now grouped in bundles
+and all management activity (deletion, upgrades, persistence) done at the level of the bundle.
+You can now start a new Brooklyn instance pointed at a persistence store,
+and it will automatically load all bundles installed into the previous Brooklyn
+through the REST API.
+
+1. Application deployment "create-or-reuse" semantics by specifying an application ID.
+This supports workflows that require something deployed in a singleton pattern,
+such as in a continuous integration pipeline.  
+
+1. jclouds latest version: improved Azure support and more.
```

Comment:
Perhaps reference the jclouds [release notes](https://jclouds.apache.org/releasenotes/2.0.2/)